app/controllers/api/v1/expertises_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Api::V1::ExpertisesController < ApplicationController
+  before_action :set_expertise, only: [:show, :update, :destroy]
+  
+  def index
```

Comment:
Should we return a message if there are no expertises associated with this user?

Reply:
Thanks for the suggestion. Have revised this in the latest commit to show error message